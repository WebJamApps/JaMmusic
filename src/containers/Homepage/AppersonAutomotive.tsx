import React from 'react';

const AppersonAutomotive = (): JSX.Element => (
  <div className="elevation2 project apperson">
    <h4>Apperson Automotive</h4>
    <div>
      <img
        alt="Apperson Automotive Home Page"
        src="../static/imgs/AppersonAutomotive.png"
      />
      <br />
      <a
        href="https://www.appersonautomotive.com"
        target="_blank"
        rel="noreferrer noopener"
      >
        appersonautomotive.com
      </a>
    </div>
  </div>
);

export default AppersonAutomotive;
