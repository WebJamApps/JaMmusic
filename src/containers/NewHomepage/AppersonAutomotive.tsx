
function AppersonAutomotive(): JSX.Element {
  return (
    <div
      // eslint-disable-next-line max-len
      style={{ boxShadow: '0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.4)', height: '500px' }}
    >
      <div>
        <img
          style={{ width: '95%', height: '455px' }}
          alt="Apperson Automotive Home Page"
          src="../static/imgs/AppersonAutomotive.png"
        />
        <br />
        <a
          href="https://www.appersonautomotive.com"
          target="_blank"
          rel="noreferrer noopener"
        >
          Appersonautomotive.com
        </a>
      </div>
    </div>
  );
}

export default AppersonAutomotive;
