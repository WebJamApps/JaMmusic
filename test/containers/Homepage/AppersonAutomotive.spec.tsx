import React from 'react';
import { shallow } from 'enzyme';
import AppersonAutomotive from '../../../src/containers/Homepage/AppersonAutomotive';

const wrapper = shallow<typeof AppersonAutomotive>(<AppersonAutomotive/>);

describe('AppersonAutomotive', () => {
  it('Renders the AppersonAutomotive', () => {
    expect(wrapper.find('div.elevation2.project.apperson').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
