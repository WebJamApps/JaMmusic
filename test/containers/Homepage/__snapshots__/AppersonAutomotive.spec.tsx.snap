// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppersonAutomotive Renders the AppersonAutomotive 1`] = `ShallowWrapper {}`;
